attributes/default.rb

```diff
@@ -99,6 +99,7 @@
 #
 default['mariadb']['galera']['cluster_name'] = 'galera_cluster'
 default['mariadb']['galera']['cluster_search_query'] = ''
+default['mariadb']['galera']['cluster_replication_port'] = '4567'
 # All Galera nodes should get the same server_id
```

Comment:
This attribute isn't used to configure what port to listen on for replication traffic - is something missing from this commit?